akka-typed-tests/src/test/java/jdocs/akka/typed/GracefulStopDocTest.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package jdocs.akka.typed;
+
+//#imports
+import akka.typed.ActorSystem;
+import akka.typed.Behavior;
+import akka.typed.PostStop;
+import akka.typed.javadsl.Actor;
+//#imports
+import java.util.concurrent.TimeUnit;
+import scala.concurrent.Await;
+import scala.concurrent.duration.Duration;
+
+public class GracefulStopDocTest {
+
+    //#master-actor
+    public abstract static class JobControl {
+        //no instances of this class, it's only a name space for messages
+        // and static methods
```

Comment:
method inside? missing space before `no`?

Reply:
fixed missing space. Didn't exactly get the point about "method inside". Would you be so kind and clarify?